mbedtls-sys/build/bindgen.rs

```diff
@@ -116,6 +118,10 @@ impl super::BuildConfig {
             };
         }
 
+        if let Some(target) = env::var_os("RUST_MBEDTLS_BINDGEN_TARGET"){
+            env::set_var("TARGET", target);
```

Comment:
I'm curious, what are you setting `RUST_MBEDTLS_BINDGEN_TARGET` to and what is `TARGET` originally set to before you set it?

Reply:
In the enclaves, with https://github.com/mobilecoinfoundation/mobilecoin/pull/1743, this is being set to 
```
RUST_MBEDTLS_BINDGEN_TARGET = "x86_64-unknown-linux-gnu"
```
Without this _override_ bindgen tries to use the `TARGET` of the enclaves:
```
  TARGET = Some("x86_64-mobilecoin-none-sgx")
```
This will result in a bindgen error of
```
  /usr/include/limits.h:26:10: fatal error: 'bits/libc-header-start.h' file not found
  /usr/include/limits.h:26:10: fatal error: 'bits/libc-header-start.h' file not found, err: true
  thread 'main' panicked at 'bindgen error: ()', /home/nick/git/rust-mbedtls/mbedtls-sys/build/bindgen.rs:146:14
```
This is because `x86_64-mobilecoin-none-sgx` will be passed to clang as the target triple.  Clang will populate the include path based on the target triple, however there is no `x86_64-none-sgx` directory on the system, there is a `/usr/include/x86_64-linux-gnu` which has `/usr/include/x86_64-linux-gnu/bits/libc-header-start.h`

Reply:
I see, thanks for the explanation. Might be worth putting this in a comment above this stuff to explain why it's necessary.